docs/yaml/kubectl/kubectl_run.yaml

```diff
@@ -25,7 +25,7 @@ options:
     If true, a public, external service is created for the container(s) which are run
 - name: generator
   usage: |
-    The name of the API generator to use.  Default is 'deployment/v1beta1' if --restart=Always, otherwise the default is 'job/v1'.  This will happen only for cluster version at least 1.2, for olders we will fallback to 'run/v1' for --restart=Always, 'run-pod/v1' for others.
+    The name of the API generator to use.  Default is 'deployment/v1beta1' if --restart=Always, 'job/v1' for OnFailure and 'pod/v1' for Never.  This will happen only for cluster version at least 1.2, for olders we will fallback to 'run/v1' for --restart=Always, 'run-pod/v1' for others.
```

Comment:
Hm, but we don't support "`restart=Never` creates pods" in 1.2 (we're adding this in 1.3). I'd however love to keep this succinct...